mw-webapp/src/logic/wayPage/wayStatistics/WayStatistic.tsx

```diff
@@ -85,13 +85,12 @@ export const WayStatistic = (props: WayStatisticProps) => {
     return null;
   }
 
-  const allDatesTimestamps = props.dayReports.map(report => report.createdAt.getTime());
-  const maximumDateTimestamp = Math.max(...allDatesTimestamps);
-  const minimumDateTimestamp = Math.min(...allDatesTimestamps);
+  const currentDate = new Date();
+  const allReports = props.dayReports;
```

Comment:
there is no guarantee that the last element will be the oldest one. Also you changed the logic. Now all abandoned and completed ways  will show wrong statistics. So please base your calculations on minimum and maximum timestamps, please

Reply:
Also please use Math.ceil instead of Nath.round, because it will be strange to see on evening that you already spent 2 days if you just created a one report in morning.

Reply:
Done